zerver/views/typing.py

```diff
@@ -14,6 +15,21 @@ def send_notification_backend(
         request: HttpRequest,
         user_profile: UserProfile,
         operator: str=REQ('op'),
-        notification_to: List[int]=REQ('to', validator=check_list(check_int))) -> HttpResponse:
-    check_send_typing_notification(user_profile, notification_to, operator)
+        notification_to: Optional[List[int]]=REQ('to', validator=check_list(check_int), default=None),
+        to_stream: Optional[int]=REQ('stream_id', validator=check_int, default=None),
+        to_topic: Optional[str]=REQ('topic', default=None)) -> HttpResponse:
+
+    if not any([notification_to, to_stream, to_topic]):
+        return json_error(_("Insufficient arguments. Should have 'to' or both 'stream_id' and 'topic'."))
+
+    if all([notification_to, to_stream, to_topic]):
+        return json_error(_("All 'to', 'stream_id', and  'topic' at once are not accepted"))
```

Comment:
I think this should probably be the same error message as the previous one, to avoid making our translators do extra strings for no reason.

Reply:
Changed all errors regarding this to "Bad arguments. Should have 'to' or both 'stream_id' and 'topic'."